parity/cli/mod.rs

```diff
@@ -353,6 +363,13 @@ struct Mining {
 }
 
 #[derive(Default, Debug, PartialEq, RustcDecodable)]
+struct Stratum {
+	interface: Option<String>,
```

Comment:
IMHO `disabled` should be part of the config - What If I want to use defaults for everything (I know I can put empty `[stratum]` but it's not in-line with other options)? 
What if I want to disable the service but keep other parameters (I know I can comment out everything, but it's not in-line with other options and it's more ugly then just switching a flag)